spec/support/matchers/matchers.rb

```diff
@@ -70,7 +70,7 @@ def actual_row(report, row, row_idx)
 RSpec::Matchers.define(:match_user_facing_csv) do |expected|
   match do |actual|
     doc = "\xEF\xBB\xBF" + expected
-    doc.to_s == actual
+    doc.to_s.gsub("\r\n", "\n") == actual.gsub("\r\n", "\n")
```

Comment:
This sets off some alarm bells for me. Why is this needed? I'd prefer to match line endings exactly. We need to have control over those for reasons mentioned above. It was not an accident that we had non-standard line endings in some of those CSV files before... Help me understand what happened here?

Reply:
Basically these specs pass locally but fail on Travis without the `gsub` here. I am unable to actually commit proper newlines because git converts them automatically.

It may be configurable with a local git config, I will look into this. I didn't realize the endings were so critical to Excel, so this clearly isn't a good solution.

Reply:
Fixed with `git config --local core.autocrlf false`; I also rebased and excluded the old commits from history.